content/posts/2021/reana-0.7.2.md

```diff
@@ -0,0 +1,88 @@
+---
+title: "REANA 0.7.2 is released"
+date: 2021-02-04T14:46:00+01:00
+tags:
+  - release
+---
+
+We are glad to announce the release of REANA 0.7.2, a minor update
```

Comment:
FYI, just a WIP draft for illustration.. The text is to be improved still, as well as the layout, e.g. center images, e.g. dunno about black background of screenshots, e.g. dunno about headers in different coulour than the main title...